buildscripts/executeUnitTests.sh

```diff
@@ -130,17 +132,96 @@ cp index.html $WORKSPACE/reports/index.html
 mkdir -p $WORKSPACE/ux-aspects/reports
 cp UXAspectsTestsResults.html reports/index.html
 
-# echo Listing WORKSPACE
-# ls -alR $WORKSPACE
-# echo Listing /home/jenkins/jobs/HPElements/jobs/ux-aspects-unit-tests
-# ls -alR /home/jenkins/jobs/HPElements/jobs/ux-aspects-unit-tests
-
 # Test for success i.e. zero failures. If there were failures, exit with status 1.
 if grep -q  ">> 0 failures" UnitTestResults.txt;
 then
     echo Unit tests passed
-    exit 0
 else
     echo "Unit test(s) failed"
     exit 1
 fi
+
+# Clear up previous documentation builds
+rm -rf $WORKSPACE/ux-aspects/KeppelThemeFiles
```

Comment:
This file is named executeUnitTests.sh - this part ought to be in a file named after what it does. I'd suggest making a continousIntegration master script which calls scripts for each discrete task that it performs.